docs/lib/datastore/fragments/android/data-access.md

```diff
@@ -1,6 +1,13 @@
 ## Save Data
 
-To write any data to the DataStore, you can pass an instance of a Model to `DataStore.save()` and it will be persisted in offline storage. The DataStore supports standard storage operations like querying, updating and deleting. If you choose to connect to the cloud at a later time, the local items will be synchronized using GraphQL mutations. Any other systems connected to the same backend can then run queries or mutations on your DataStore items, or observe changes to them via GraphQL subscriptions.
+To write data to the DataStore, you can pass an instance of a `Model` to
```

Comment:
I prefer editing Markdown like this also at 80 columns, but the convention of this repo is long lines so we should conform to that.